docs/windows_installation.md

```diff
@@ -0,0 +1,49 @@
+We **strongly** recommend that Windows users use [Docker](docker.md) as this will work much easier and smoother (also more secure).
+
+If that is not possible, try using the Windows Linux subsystem (WSL) - for which the Ubuntu instructions should work.
+Otherwise, try the instructions below.
+
+## Install freqtrade manually
+
+!!! Note
+    Make sure to use 64bit Windows and 64bit Python to avoid problems with backtesting or hyperopt due to the memory constraints 32bit applications have under Windows.
+
+!!! Hint
+    Using the [Anaconda Distribution](https://www.anaconda.com/distribution/) under Windows can greatly help with installation problems. Check out the [Conda section](#using-conda) in this document for more information.
+
+### 1. Clone the git repository
+
+```bash
+git clone https://github.com/freqtrade/freqtrade.git
+```
+
+### 2. Install ta-lib
+
+Install ta-lib according to the [ta-lib documentation](https://github.com/mrjbq7/ta-lib#windows).
+
+As compiling from source on windows has heavy dependencies (requires a partial visual studio installation), there is also a repository of unofficial precompiled windows Wheels [here](https://www.lfd.uci.edu/~gohlke/pythonlibs/#ta-lib), which needs to be downloaded and installed using `pip install TA_Lib‑0.4.18‑cp38‑cp38‑win_amd64.whl` (make sure to use the version matching your python version)
```

Comment:
Note to self:
this section should be updated - the wheel files are included in the build-helper directory, so seperate downloading is not necesary

Reply:
I will let you do your thing here because I tried to install on Windows and "Oh boy..."